phsa-adapter/src/AuthenticatedUser.cs

```diff
@@ -1,21 +0,0 @@
-﻿using System.Security.Principal;
-
-namespace Rsbc.Dmf.PhsaAdapter
-{
-    // AuthenticatedUser.cs
```

Comment:
The basic auth code is needed for the non-FHIR inbound payload from PHSA

Reply:
restored